.github/workflows/build.yml

```diff
@@ -13,7 +13,7 @@ on:
       - '**.md'
       - '**.txt'
   schedule:
-    - cron: '20 19 * * *'  # 7:20pm everyday    
+    - cron: '0 5 * * *'  # 5 am everyday
```

Comment:
Eh I asked about the 7am in another place: https://github.com/ublue-os/main/pull/87#discussion_r1152663602

But same question as this would leave only 1 hour of buffer after the 4am UTC completion of builds.

Reply:
Question was answered in Discord and this seems good to me.